fortnite/README.md

```diff
@@ -0,0 +1,142 @@
+# Fortnite
+
+Provides performance insights and metrics tracking for both professional and amateur Fortnite players such as friends and family using the unofficial [Fortnite API](https://fortniteapi.io/) service.
+
```

Comment:
Can you add a `Quick Install` section here, based on https://github.com/influxdata/community-templates/blob/master/Example_README.md#quick-install

Reply:
Added Quick Install section.